.coveragerc

```diff
@@ -24,3 +24,6 @@ exclude_lines =
 
     # Don't complain if non-runnable code isn't run:
     if __name__ == .__main__.:
+
+    # Don't complain if ellipsis never gets executed
+    grep -r "^[ ]*\.\.\.$"
```

Comment:
Confused about the `grep -r` part, is this just a bad paste?

Reply:
Oops! Yes, will fix that along with the typedef thing